Makefile

```diff
@@ -82,6 +82,14 @@ ifeq ($(DISABLE_CGO), 1)
 	override BUILDTAGS = exclude_graphdriver_devicemapper exclude_graphdriver_btrfs containers_image_openpgp
 endif
 
+ARCHES=amd64 s390x ppc64le
+MA_IMAGE := $(REGISTRY)/$(REPO):$(TAG)
```

Comment:
`MA` is non-obvious. Please use something more explicit, maybe just `MULTIARCH` or `MULTI_ARCH`. (everywhere)

Reply:
Used `MULTIARCH` everywhere